news/bugfix-3398.md

```diff
@@ -0,0 +1 @@
+`syslog-ng`: fix spelling issues
```

Comment:
I would extend this like: "fixed numerous spelling mistakes in messages generated by syslog-ng"